eth/tracers/js/goja.go

```diff
@@ -238,7 +250,7 @@ func (t *jsTracer) CaptureStart(env *vm.EVM, from common.Address, to common.Addr
 	t.ctx["block"] = t.vm.ToValue(env.Context.BlockNumber.Uint64())
 	// Update list of precompiles based on current block
 	t.activePrecompiles = env.ActivePrecompiles()
-	t.ctx["intrinsicGas"] = t.vm.ToValue(t.gasLimit - gas)
+	t.ctx["intrinsicGas"] = t.vm.ToValue(t.gasLimit - gas) // PTAL-meowsbits Is this supposed to be here?
```

Comment:
Yes at the moment as tracers need this to be available.

```suggestion
	t.ctx["intrinsicGas"] = t.vm.ToValue(t.gasLimit - gas)
```

Reply:
Indeed this can go now. But some stuff have to be changed on some tracers as well. I will make a commit to fix this one

Reply:
Resolved by https://github.com/etclabscore/core-geth/pull/531/commits/a253f932f68cc444c36dffe2ceb2f1e5c3935571